docs/SPEC.md

```diff
@@ -3338,7 +3346,82 @@ Example of script:
 // A 5 point double exponential moving average would be called as such:
 from(bucket: "telegraf/autogen"):
     |> range(start: -7d)
-    |> doubleExponentialMovingAverage(n: 5, columns: ["_value"])
+    |> doubleEMA(n: 5)
+```
+
+#### Triple Exponential Moving Average
+`tripleEMA`
+
+Triple Exponential Moving Average computes the triple exponential moving averages of the `_value` column.
+It is a weighted moving average that gives more weighting to recent data as opposed to older data, with less lag than a double or single exponential moving average.
+
+| Name        | Type     | Description
+| ----        | ----     | -----------
+| n           | int      | N specifies the number of points to mean.
+
+A triple exponential moving average is defined as `TEMA = (3 * EMA_1) - (3 * EMA_2) + EMA_3`, where
+- `EMA_1` is the exponential moving average of the original data,
+- `EMA_2` is the exponential moving average of `EMA_1`,
+- `EMA_3` is the exponential moving average of `EMA_2`, and
+- `N = n` is the look-back period.
+
+In flux, `tripleEMA` is defined as
+```
+tripleEMA = (n, tables=<-) =>
+	tables
+		|> exponentialMovingAverage(n:n)
+		|> duplicate(column:"_value", as:"ema1")
```

Comment:
Defining temporary columns like this seems potentially dangerous, since `duplicate` will silently overwrite data if the new column already exists.  It's unlikely, but a user might already have a column called `ema1`, etc.

It seems like we need a convention for column names that are only used internally.  That would require some more extensive language/library design, but for the time being, maybe change these temporary columns to `__ema1`, `__ema2`, and so on?